angular.json

```diff
@@ -123,6 +128,7 @@
           }
         }
       }
-    }},
+    }
+  },
   "defaultProject": "chuck-jokes-app"
-}
+}
```

Comment:
GitHub wants empty end line for each file.

Reply:
Fixed